wolfssl/wolfcrypt/ed25519.h

```diff
@@ -90,6 +93,9 @@ struct ed25519_key {
 #if defined(WOLF_CRYPTO_CB)
     int devId;
 #endif
+    void *heap;
+    wc_Sha512 sha;
+    int sha_clean_flag;
```

Comment:
This could technically be `byte` or even`byte sha_clean_flag:1`.

Reply:
ditto